### DISCUSSION TITLE
What if there is no AppData folder?

### DISCUSSION THREAD

**Discussion by Mlrosier** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#17] [General]
I have No Man's Sky on Steam. But I don't have a HelloGames folder in my AppData Local folder under my username folder. In fact, there  is no HelloGames folder anywhere on my C drive. I have NMS installed on another drive, but there should still be a folder on my C drive in AppData shouldn't there? I added it manually, but the game doesn't recognize that there is a folder there, so none of the expeditions will load.

**Comment by cwmonkey** [comment]
> I have NMS installed on another drive, but there should still be a folder on my C drive in AppData shouldn't there?

Negative, the directory will be relative to where you installed the game.
